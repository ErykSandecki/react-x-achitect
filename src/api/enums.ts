export enum Header {
  contentType = 'Content-Type',
}
