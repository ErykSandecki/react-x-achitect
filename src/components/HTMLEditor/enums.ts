export enum BlockHeight {
  small = 50,
  medium = 70,
  large = 100,
}

export enum BlockWidth {
  small = 50,
  medium = 200,
  large = 350,
}

export enum MouseMode {
  comment = 'comment',
  default = 'default',
  menu = 'menu',
  move = 'move',
}
