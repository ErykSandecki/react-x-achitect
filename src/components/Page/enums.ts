export enum PageType {
  blank = 'blank',
  edit = 'edit',
  layout = 'layout',
}
