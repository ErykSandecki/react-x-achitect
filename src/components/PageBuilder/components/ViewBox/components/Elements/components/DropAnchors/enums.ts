export enum DropAnchorsPosition {
  top = 'top',
  bottom = 'bottom',
  left = 'left',
  right = 'right',
}
