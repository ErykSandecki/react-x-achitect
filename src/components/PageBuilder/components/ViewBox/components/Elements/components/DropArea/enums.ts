export enum DropAreaPosition {
  top = 'top',
  bottom = 'bottom',
  left = 'left',
  right = 'right',
}
