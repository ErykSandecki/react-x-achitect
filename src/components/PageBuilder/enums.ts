export enum Constrain {
  bottom = 'bottom',
  centerHorizontal = 'centerHorizontal',
  centerVertical = 'centerVertical',
  left = 'left',
  right = 'right',
  top = 'top',
}
