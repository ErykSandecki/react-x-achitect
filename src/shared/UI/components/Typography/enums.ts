export enum TypographyColor {
  blue1 = 'blue1',
  green1 = '28a745',
  green2 = 'a7b45d',
  neutral1 = 'neutral1',
  neutral2 = 'neutral2',
  neutral3 = 'neutral3',
  neutral4 = 'neutral4',
  neutral5 = 'neutral5',
  orange1 = 'orange1',
  orange2 = 'orange2',
  pink1 = 'pink1',
  red1 = 'red1',
  skyBlue1 = 'skyBlue1',
  violet1 = 'violet1',
}

export enum TypographyFontStyle {
  italic = 'italic',
  normal = 'normal',
}

export enum TypographyFontType {
  h1 = 'h1',
  h2 = 'h2',
  h3 = 'h3',
  h4 = 'h4',
  h5 = 'h5',
  h6 = 'h6',
  p = 'p',
  small = 'small',
}

export enum TypographyFontWeight {
  blacks = 'blacks',
  bold = 'bold',
  extraBold = 'extra-bold',
  light = 'light',
  medium = 'medium',
  regular = 'regular',
  semiBold = 'semi-bold',
}
