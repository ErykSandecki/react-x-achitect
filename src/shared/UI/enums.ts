export enum InputSize {
  small = 'small',
  medium = 'medium',
  large = 'large',
}
