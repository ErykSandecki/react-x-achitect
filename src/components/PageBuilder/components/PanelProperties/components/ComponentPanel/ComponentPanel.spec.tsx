import { Provider } from 'react-redux';
import { render } from '@testing-library/react';

// components
import ComponentPanel from './ComponentPanel';

// store
import { configureStore } from 'store/store';

// types
import { Tab } from '../../enums';

describe('ComponentPanel snapshots', () => {
  it('should render ComponentPanel', () => {
    // mock
    const store = configureStore();

    // before
    const { asFragment } = render(
      <Provider store={store}>
        <ComponentPanel activeTab={Tab.design} />
      </Provider>,
    );

    // result
    expect(asFragment()).toMatchSnapshot();
  });

  it('should render proptype section', () => {
    // mock
    const store = configureStore();

    // before
    const { asFragment } = render(
      <Provider store={store}>
        <ComponentPanel activeTab={Tab.prototype} />
      </Provider>,
    );

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
