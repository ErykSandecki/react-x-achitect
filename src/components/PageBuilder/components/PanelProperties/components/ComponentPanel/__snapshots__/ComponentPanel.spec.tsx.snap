// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentPanel snapshots should render ComponentPanel 1`] = `
<DocumentFragment>
  <div
    class="css-q0izro Section Section--light"
    data-test-section=""
  >
    <div
      class="css-q0izro SectionColumn SectionColumn--light"
      data-test-section=""
    >
      <div
        class="SectionColumn__content SectionColumn__content--light"
      >
        <small
          class="css-q0izro Typography Typography--light Typography--neutral1 Typography--neutral1--light Typography--regular Typography--regular--light Typography--normal Typography--normal--light Typography--small Typography--small--light"
          data-test-text=""
          style="text-align: inherit;"
        >
          0 selected
        </small>
      </div>
      <div
        class="SectionColumn__buttons SectionColumn__buttons--light"
      >
        <div
          class="css-q0izro ButtonIcon ButtonIcon--light"
          data-test-button-icon=""
        >
          <div
            class="css-q0izro Icon Icon--light Icon--neutral1 Icon--neutral1--light"
            data-test-icon="HtmlTag"
            height="14"
            name="HtmlTag"
            width="14"
          />
        </div>
        <div
          class="css-q0izro ButtonIcon ButtonIcon--light"
          data-test-button-icon=""
        >
          <div
            class="css-q0izro Icon Icon--light Icon--neutral1 Icon--neutral1--light"
            data-test-icon="Component"
            height="14"
            name="Component"
            width="14"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`ComponentPanel snapshots should render proptype section 1`] = `<DocumentFragment />`;
